src/fillTank.test.js

```diff
@@ -1,11 +1,147 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should ', () => {});
 
+  // write tests here
+
+  it(`should be declared`, () => {
+    expect(fillTank).toBeInstanceOf(Function);
   });
 
-  // write tests here
+  it(`should return nothing`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    expect(fillTank(customer)).toBeUndefined();
+  });
+
+  it(`should ordered full tank if amount is not given`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 10,
+      },
+    };
+
+    fillTank(customer, 5);
+
+    expect(customer).toEqual({
+      money: 2850,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 40,
+      },
+    });
+  });
```

Comment:
The test named should ordered full tank if amount is not given does not actually handle the "full tank" scenario if no amount is passed. It calls `fillTank(customer, 5)`

Reply:
I do not understand you, default value for amount is Infinity